src/ALCProxy.Communication/ALCProxy.Communication.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
Is this meant to be an exe or a library (i.e. dll)? At a quick glance, this looks like a library to me so we probably don't want this `OutputType`. Please ignore this feedback if I am mistaken.

Otherwise, consider looking at other csproj files within the repo for libraries, like:
https://github.com/dotnet/corefxlab/blob/4cefa703d08c2ddc8f950d9ca211cfde06c8f484/src/System.Collections.Sequences/System.Collections.Sequences.csproj#L2-L8